test-registry.yaml

```diff
@@ -19,4 +19,4 @@ metadata:
 spec:
   sourceType: grpc
   displayName: Cockroach Test Demo
-  image: quay.io/alinalion/cockroach-operator-bundle:my-index
```

Comment:
What is this file for?

Reply:
This file  is for test in openshift installation